schemas/0.0.1-preview/CreateUIDefinition.CommonControl.json

```diff
@@ -459,7 +459,13 @@
           "properties": {
             "required": {
               "type": [ "boolean", "string" ]
-            }
+            },
```

Comment:
Is this actually supported properly for other elements? I"m worried that there may be at least one element where this behavior won't work as expected if we push this schema change.